docs/dashboard/anomaly-advisor-tab.md

```diff
@@ -0,0 +1,28 @@
+# Anomaly Advisor tab
+
+The Anomaly Advisor tab lets you quickly surface potentially anomalous metrics and charts related to a particular highlighted window of interest.
+
+To enable the Anomaly Advisor you must first ensure that ML is enabled on your nodes.
+
+More details about configuration can be found in the [ML documentation](https://github.com/netdata/netdata/blob/master/src/ml/README.md#configuration).
+
+This tab uses our [Anomaly Rate ML feature](https://github.com/netdata/netdata/blob/master/src/ml/README.md#anomaly-rate---averageanomaly-bit) in order to score metrics in terms of anomalous behavior.
+
+- The "Anomaly Rate" chart shows the percentage of anomalous metrics over time per node.
+
+- The "Count of Anomalous Metrics" chart shows raw counts of anomalous metrics per node so may often be similar to the Anomaly Rate chart, apart from where nodes may have different numbers of metrics.
+
+- The "Anomaly Events Detected" chart shows if the anomaly rate per node was sufficiently elevated to trigger a node level anomaly. Anomaly events will appear slightly after the anomaly rate starts to increase in the timeline, this is because a significant number of metrics in the node need to be anomalous before an anomaly event is triggered.
```

Comment:
```suggestion
- The "Anomaly Events Detected" chart shows whether the anomaly rate per node has increased enough to cause a node-level anomaly. Anomaly events will appear slightly after the anomaly rate starts to increase in the timeline, this is because a significant number of metrics in the node need to be anomalous before an anomaly event is triggered.
```